packages/@lwc/synthetic-shadow/src/polyfills/window-event-target/README.md

```diff
@@ -0,0 +1,11 @@
+# window event target polyfill
+
+This polyfill is specifically for IE11. Please remove if we ever stop supporting it.
+
+In all the modern browsers we support, `addEventListener()`, `removeEventListener()`, and
+`dispatchEvent()` are defined on the `EventTarget` interface. In IE11, these methods are defined
+on the `Node` interface, and so our event target polyfill ends up patching `Node.prototype` in
+the case that `EventTarget` is not defined.
+
+IE11 throws us yet another curveball where `window` is not an instance of `Node`, so we cover our
```

Comment:
`window` is never an instance of `Node`. It should be that being said an instance of `EventTarget`. Is it the issue with IE11?